scripts/publish.sh

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+TOP=$PWD
+
+# This is the long-living fork of upstream we use to generate the PR
+# from.
+FORK=SUSE/cli-plugin-repo
+
+HUB_RELEASE=https://github.com/github/hub/releases/download/v2.3.0-pre10/hub-linux-amd64-2.3.0-pre10.tgz
```

Comment:
Can this be a Concourse Github release resource? That way it could be cached between runs, and we could always use the latest version.